### PR TITLE
matts branch

### PR DESCRIPTION
**Pull request by denty8**
and added story

### CODE REVIEW

File: 29th-aug/story.md [hunk 1]
```diff
@@ -2,4 +2,5 @@ Once upon a time there was a pride of WfT analysts
 The pride was full of love and smiles, until...
 They were set upon by a fever of sting-rays
 Who shouted out loud;
-That they were hungry
\ No newline at end of file
+That they were hungry
+But they were nice rays and really, they just wanted a hug
```
Comment (kserevena) [inline 29th-aug/story.md]: yeah they were!

Comment (chloegillham) [reply 29th-aug/story.md]: Rays of sunshine <3

Comment (denty8) [reply 29th-aug/story.md]: glad we all agree

**Review by kserevena — APPROVED** [review]
good to merge!

**Pull request merged**